src/timedate/timedatectl.c

```diff
@@ -261,6 +261,11 @@ static int set_local_rtc(int argc, char **argv, void *userdata) {
         if (b < 0)
                 return log_error_errno(b, "Failed to parse local RTC setting '%s': %m", argv[1]);
 
+       if (b == 1) {
+               fflush(stdout);
+               log_warning("Warning: The system is now being configured to read the RTC time in the local time zone. This mode cannot be fully supported. It will create various problems with time zone changes and DST adjustments. If at all possible, use RTC in UTC.");
```

Comment:
Hmm, why are linebreaks dropped? Please don't output a large chunk of text in a single line.

Reply:
Hmm, why? I think @keszybz's version is better. I just wanted to retain the linebreaks to ensure we're not dumping a bunch of text as a single line.

Reply:
> Hmm, why? I think @keszybz's version is better. I just wanted to retain the linebreaks to ensure we're not dumping a bunch of text as a single line.

 @keszybz's version only suggested to remove the command from warning which I did in latest commit. I think complete warning makes sense here as it tells the possible consequences. I think user might neglect the shorter version.

Reply:
Nah, what I'm referring to is "being configured" vs "configured"

Reply:
Oh sorry missed that part. I will correct it.